blockchain_connector/avalon_blockchain_connector/connector_service.py

```diff
@@ -96,8 +99,12 @@ def main(args=None):
             logging.info("About to start Ethereum connector service")
             eth_connector_svc = EthereumConnector(config)
             eth_connector_svc.start()
-        else:
+        elif proxy_blockchain == FABRIC:
             logging.info("About to start Fabric connector service")
+            fabric_connector_svc = FabricConnector(uri)
+            fabric_connector_svc.start()
+        else:
```

Comment:
Can be removed. The argparser takes care of this.

Reply:
Done.